packs/feats/lingering-flames.json

```diff
@@ -31,7 +31,64 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: The Grand Bazaar"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.SpellTrickster.Label",
+                "mergeable": true,
+                "option": "spell-trickster",
+                "placement": "spellcasting",
+                "suboptions": [
+                    {
+                        "label": "{item|name}",
+                        "value": "lingering-flames"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "spell-trickster:lingering-flames",
+                    "item:slug:fireball"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.SpellTrickster.LingeringFlames.Description"
+                    },
+                    {
+                        "divider": true,
+                        "text": "PF2E.SpecificRule.SpellTrickster.LingeringFlames.DescriptionHeightened",
+                        "title": "PF2E.SpecificRule.SpellHeightened.HeightenedPlusOne"
+                    }
+                ]
+            },
+            {
+                "key": "DamageDice",
```

Comment:
Consider using a DamageAlteration instead.

Reply:
I'm getting a warning with the DamageAlteration I need, which may be a bug, so I'll hold off on using it until that's resolved.

Reply:
The shark moves fast. It's fixed, using DamageAlteration now!